examples/fully-private-eks-cluster/eks/main.tf

```diff
@@ -0,0 +1,57 @@
+provider "aws" {
+  region = var.region
+  alias  = "default"
+}
+
+provider "kubernetes" {
+  host                   = module.eks_blueprints.eks_cluster_endpoint
+  cluster_ca_certificate = base64decode(module.eks_blueprints.eks_cluster_certificate_authority_data)
+
+  exec {
+    api_version = "client.authentication.k8s.io/v1alpha1"
+    command     = "aws"
+    # This requires the awscli to be installed locally where Terraform is executed
+    args = ["eks", "get-token", "--cluster-name", module.eks_blueprints.eks_cluster_id]
+  }
+}
+
+locals {
+  name               = basename(path.cwd)
+  vpc_id             = var.vpc_id
+  private_subnet_ids = var.private_subnet_ids
+  tags = {
+    Blueprint  = local.name
+    GithubRepo = "github.com/aws-ia/terraform-aws-eks-blueprints"
+  }
+}
+
+module "eks_blueprints" {
+  source = "../../.."
+
+  cluster_name = local.name
+
+  # EKS Cluster VPC and Subnets
+  vpc_id             = local.vpc_id
+  private_subnet_ids = local.private_subnet_ids
+
+  # Cluster Security Group
+  cluster_security_group_additional_rules = var.cluster_security_group_additional_rules
+
+  # EKS CONTROL PLANE VARIABLES
+  cluster_version = var.cluster_version
+
+  cluster_endpoint_public_access  = false
```

Comment:
Change this to `true` and add notes to changed to `false` once the cluster is deployed. Also, advise users to use Bastion host to communicate with cluster once its changed to false